File: CityDangersAlertWeb/src/app/Models/Problem.ts
export class Problem
{
    partitionKey: string;
    rowKey: string;
    user: string;
    status: string;

    constructor(type: string, location: string, user: string, status: string){
        this.partitionKey = type;
        this.rowKey = location;
        this.user = user;
        this.status = status;
    }
}

File: CityDangersAlert/app/src/main/java/com/example/citydangersalert/Problem.java
package com.example.citydangersalert;

import androidx.annotation.Nullable;

import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.Marker;

public class Problem {
    public String user;
    public String status;
    public String type;//partitionkey
    public LatLng position;//rowkey
    public Marker marker;
}
